Sources/Support/POSIX/Stringify.cpp

```diff
@@ -262,6 +262,14 @@ char const *Stringify::Errno(int error) {
 
 char const *Stringify::Ptrace(int code) {
   switch (code) {
+#if defined(OS_DARWIN)
```

Comment:
I just check the man PTrace on freebsd and it should work. Unfortunately freebsd build doesn't compile at this moment and I can't test :D So I will do a separate commit for freebsd PTrace stringify.